rnd/infra/helm/autogpt-server/values.dev.yaml

```diff
@@ -82,4 +82,6 @@ env:
   APP_ENV: "dev"
   PYRO_HOST: "0.0.0.0"
   NUM_GRAPH_WORKERS: 100
-  NUM_NODE_WORKERS: 100
\ No newline at end of file
+  NUM_NODE_WORKERS: 100
```

Comment:
Does this mean 100 node workers per graph worker? That seems excessive. If that's not what it means, I didn't say anything. :p

Reply:
we're just seeing how far we can stretch the system ha